packages/strickland/src/strickland.js

```diff
@@ -0,0 +1,45 @@
+import every from './every';
+import props from './props';
```

Comment:
I feel uneasy about these importing cyclically.  I understand why you did it, but I wonder if it can be avoided.